Sources/Classes/Headers/Public/RSClient.h

```diff
@@ -93,6 +93,8 @@ typedef void (^Callback)(NSObject *_Nullable);
 - (void) onIntegrationReady:(id<RSIntegrationFactory>)factory withCallback:(Callback)callback;
 - (void) clearAdvertisingId;
 
+- (void)openURL:(NSURL *)url options:(NSDictionary *)options;
```

Comment:
Let's add type safety. (You may look at other APIs e.g., track present in this class. We have declared the types for the `NSDictionary` object.)
```suggestion
- (void)openURL:(NSURL *)url options:(NSDictionary<NSString*, id>*)options;
```